cmd/subcommands/root.go

```diff
@@ -223,15 +224,21 @@ func endpointToChainID(nodeAddr string) chainIDWrapper {
 
 func validateAddress(cmd *cobra.Command, args []string) error {
 	// Check if input valid one address
-	address := oneAddress{}
-	if err := address.Set(args[0]); err != nil {
+	tmpAddr := oneAddress{}
+	if err := tmpAddr.Set(args[0]); err != nil {
 		// Check if input is valid account name
 		if acc, err := store.AddressFromAccountName(args[0]); err == nil {
 			addr = oneAddress{acc}
 			return nil
 		}
-		return fmt.Errorf("Invalid one address/Invalid account name: %s", args[0])
+
+		bech32Addr := address.ToBech32(address.Parse(args[0]))
+		if bech32Addr == "" {
+			return fmt.Errorf("Invalid one address/Invalid account name: %s", args[0])
+		}
+
+		addr = oneAddress{bech32Addr}
 	}
-	addr = address
+	addr = tmpAddr
```

Comment:
you always set addr to tmpAddr?  Then what's the usage of line 240?

Reply:
this is a bug, fixed!